Notebooks/Themes/SolveMathProblems/0-IntroductionToNumericalSolutions.md

```diff
@@ -138,44 +138,45 @@ plot(time_values, theta_values, linewidth=3.0, color="red")
 First we try fitting the local expansion for \\(\theta\\) through an
 additional point.	 
 This time we assume that the variation in \\(\theta(t)\\) is quadratic, i.e.
-$$
+\\[
     \theta(t') = \theta_{n-1} + \beta t' + \gamma {t'}^2
-$$
+\\]
 
 The local time coordinate is $t' = t - (n-1)\Delta t$, and when we differentiate
 
-$$       
+\\[
     \frac{d \theta}{dt} = \beta + 2 \gamma t'
-$$  
+\\]
 
 To solve for \\(\beta\\) and \\(\gamma\\) we fit the curve through the sample points:
-$$
+\\[
     \begin{split}
             \theta_n &= \theta_{n-1} + \beta \Delta t + \gamma (\Delta t)^2 \\
             \theta_{n+1} &= \theta_{n-1} + 2 \beta \Delta t + 4 \gamma (\Delta t)^2
-     \end{split}
-$$
+    \end{split}
+\\]
```

Comment:
btw @lmoresi, I don't think the rendering does the splitting... I see all equations in a single line...

Reply:
Yep, it's best not to be too ambitious with MathJax ... Thanks !